tests/Signal/ported.test.ts

```diff
@@ -0,0 +1,207 @@
+import {it, describe, expect, vi} from 'vitest';
+import {Signal} from '../../src';
+
+const watch = (cb) => {};
+
+describe('Ported signal tests', () => {
```

Comment:
should this instead be a folder?

like:
```js
tests/Signal/ported/
  vue.test.ts
  preactjs.test.ts
  (etc)
```

reason being is that smaller focused files are generally easier to work on (especially as each test is sort of its own subject/topic )